### PR TITLE
Improve README file

### PR DESCRIPTION
**Pull request by victormartingarcia**
## Description

Fix: #58 

Changes on README.md for better communicating what this project is, and provide some guidance to incentivize contribution

## Code formating

Only affects README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,50 @@
-## Django Job Portal
+<h1 align="center">Anem per feina</h1>
+<p align="center">
+    <img alt="forks" src="https://img.shields.io/github/forks/GeeksCAT/anem-per-feina?label=Forks&style=social"/>
+    <img alt="stars" src="https://img.shields.io/github/stars/GeeksCAT/anem-per-feina?style=social"/>
+    <img alt="watchers" src="https://img.shields.io/github/watchers/GeeksCAT/anem-per-feina?style=social"/>
+</p>
 
-#### An open source online job portal.
+**Anem per feina** is an open source project promoted by [GeeksCat](https://geekscat.org/) association for [Hactoberfest 2020](https://hacktoberfest.geekscat.org/) event.
```
Comment (XaviTorello) [inline README.md:8]: F*ck yeah!

File: README.md [hunk 2]
```diff
@@ -1,60 +1,50 @@
...
+We forked the open source project [django-job-portal](https://github.com/manjurulhoque/django-job-portal) (See [demo 1](https://django-portal.herokuapp.com/), [demo 2](http://jobs.manjurulhoque.com/)) and use it as a baseline.
 
-*Note* : Collect static is not necessary when debug is True (in dev mode)
 
-Create initial database:
+<a name="contributing"></a>
+## Contributing
 
-`python manage.py migrate`
+**Anem per feina** is a collaborative effort where everybody is more than welcome to contribute, no experience is required!
```
Comment (XaviTorello) [inline README.md:31]: 😍😍😍

File: README.md [hunk 3]
```diff
@@ -98,16 +86,59 @@ If during development you need to change the `requirements.txt` file you'll also
 
 3. Restart development containers with `docker-compose up`
```
Comment (XaviTorello) [inline README.md:87]: Should be 2

File: README.md [hunk 4]
```diff
@@ -1,60 +1,50 @@
...
+## Contributing
 
-`python manage.py migrate`
+**Anem per feina** is a collaborative effort where everybody is more than welcome to contribute, no experience is required!
 
+Have a look at [CONTRIBUTING.md](CONTRIBUTING.md) file that describes the process to submit a contribution, and come say hi 👋👋to [GeeksCat slack](https://geekscat.slack.com) where most contributors hang out.
 
-Load demo data (optional):
+<a name="tech-stack"></a>
+## Tech stack
```
Comment (XaviTorello) [inline README.md:36]: 💪💪💪

**Review by XaviTorello — CHANGES_REQUESTED** [review]
I love it @victormartingarcia, just minor list numeration change

Please, fix conflicts adding `make` references and will be merged :)

**Comment by jbagot** [comment]
I like it, for me it's approved when you will fix the comments of Xavi. Great work :) It's a fancier README

File: README.md (hunk 3 above)
Comment (victormartingarcia) [reply README.md]: Nice catch! Already fixed

**Comment by victormartingarcia** [comment]
> I love it @victormartingarcia, just minor list numeration change
> 
> Please, fix conflicts adding `make` references and will be merged :)

Fixed conflicts 👍

**Review by XaviTorello — APPROVED** [review]
💪

**Pull request merged**